src/components/profile/ProfileAvailability.jsx

```diff
@@ -81,13 +70,13 @@ const ProfileAvailability = ({ userId }) => {
       );
       const dayArray = [];
       if (dataAvailability.length > 0) {
-        const d = dataAvailability[0];
-        const startTime = d.startTime.slice(0, d.startTime.lastIndexOf(':'));
-        const endTime = d.endTime.slice(0, d.endTime.lastIndexOf(':'));
-        let available = false;
+        const startTimes = dataAvailability.map(d => {
+          return d.startTime.slice(0, d.startTime.lastIndexOf(':'));
```

Comment:
If your function is one line just move the return to the previous line; you don't need the `return` keyword